app/models/post.rb

```diff
@@ -1,3 +1,3 @@
 class Post < ApplicationRecord
-  belongs_to :users
+  belongs_to :user
```

Comment:
- [ ] The code should start by declaring a Post model with the following attributes: - name - author_id - created_at - updated_at Next, it declares that this post belongs to an Author, **hint (belong_to => author and class_name to the user);**

- [ ] It also should have many comments and likes. **hint ( has_many => comments and likes)**
- [ ] After saving, the code should increment the posts counter for the author. The update_post_counter method is private and increments a user's posts count after saving their post. The code attempts to increment the author's posts_counter every time they save a post. **hint( post.increment)**